README.md

```diff
@@ -843,6 +845,7 @@ This list results from Pull Requests, reviews, ideas, and work done by 1600+ peo
   * [letsmonitor.org](https://letsmonitor.org) - SSL monitoring, free for up to 5 monitors
   * [linkok.com](https://linkok.com) - Online broken link checker, free for small websites up to 100 pages, completely free for open-source projects.
   * [loader.io](https://loader.io/) — Free load testing tools with limitations
+  * [MetricsHub](https://metricshub.com) — MetricsHub is an OpenTelemetry collection agent that extracts metrics from any platform, application, or service using protocols like SSH, WMI, WBEM, WinRM, SNMP, or HTTP. It seamlessly pushes these metrics to any observability platform that supports OpenTelemetry, such as Datadog, Prometheus, Grafana, Splunk, or New Relic. MetricsHub is available in two editions: Community, for basic infrastructure monitoring, and Enterprise, offering full coverage for advanced monitoring needs.
```

Comment:
few issuers here - but primarily it does not seem to me like a SaaS offering, you install the agent locally and the agent forwards to wherever you tell it.  No SaaS involved?

I don't really see it clearly in the docs to figure out if this is SaaS involved or not.

Reply:
Yes you're right @ripienaar, it is not a SaaS offering. Users install the agent locally and push the collected metrics to the SaaS observability platform of their choice.

Reply:
Thanks, will close this then as this is a list for SaaS software.